wallet/src/de/schildbach/wallet/ui/dashpay/CropImageActivity.kt

```diff
@@ -89,14 +89,15 @@ class CropImageActivity : LockScreenActivity() {
                         background.setImageDrawable(resource)
                         circle_crop.setImageDrawable(resource)
                         circle_crop.maxZoom = 5f
-                        if (initZoomedRect != null) {
+
+                        initZoomedRect?.let { rect ->
```

Comment:
kotlin 1.5 strick null-safety (`initZoomedRect` is lazy and so a simple null-check doesn't work).